addons/hr_holidays/models/hr_leave_accrual_plan.py

```diff
@@ -10,11 +10,15 @@ class AccrualPlan(models.Model):
 
     name = fields.Char('Name', required=True)
     time_off_type_id = fields.Many2one('hr.leave.type', string="Time Off Type",
+        check_company=True,
         help="""Specify if this accrual plan can only be used with this Time Off Type.
                 Leave empty if this accrual plan can be used with any Time Off Type.""")
     employees_count = fields.Integer("Employees", compute='_compute_employee_count')
     level_ids = fields.One2many('hr.leave.accrual.level', 'accrual_plan_id', copy=True)
     allocation_ids = fields.One2many('hr.leave.allocation', 'accrual_plan_id')
+    company_id = fields.Many2one('res.company', string='Company',
+        compute="_compute_company_id", store="True",
+        readonly=False, default=lambda self: self.env.company)
```

Comment:
```suggestion
        compute="_compute_company_id", store=True,
        readonly=False)
```

Reply:
Don't set a default value on a compute, instead you can set the default value you want in the compute itself